operations/app/src/modules/nat_gateway/main.tf

```diff
@@ -0,0 +1,37 @@
+terraform {
+  required_version = ">= 0.14"
+}
+
+resource "azurerm_public_ip" "nat_gateway_ip" {
```

Comment:
Could we add a [`lifecycle.prevent_destroy = true`](https://www.terraform.io/docs/language/meta-arguments/lifecycle.html) to this module? This contains data that would be damaging if accidentally ran a terraform destroy on it.

Reply:
Added